ArmRobot/Assets/Scripts/Camera/RandomizerPositionCamera.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class RandomizerPositionCamera : MonoBehaviour
+{
+    public float changePositionCamera = 0.005f;
+    public float initialCameraPositionXAxis = 0.0f;
+    public float initialCameraPositionYAxis = 1.8f;
+    public float initialCameraPositionZAxis = 0.0f;
```

Comment:
You could use Vector3 instead of a separate variable for x, y, and z. That way you could replace these three lines with just `public Vector3 initialCameraPosition = new Vector3(0.0f, 1.8f, 0.0f);`

Reply:
Or better yet - leave it blank `public Vector3 initialCameraPosition` and set it in the `Start()` method from the current position. That way if the user wants to move the camera, they can just move it in the scene, and do not have to separately copy these numbers in.